pyface/tasks/task_window.py

```diff
@@ -454,9 +457,9 @@ class TaskState(HasStrictTraits):
 
     central_pane = Instance(ITaskPane)
     dock_panes = List(IDockPane)
-    menu_bar_manager = Instance(MenuBarManager)
-    status_bar_manager = Instance(StatusBarManager)
-    tool_bar_managers = List(ToolBarManager)
+    menu_bar_manager = Instance(IMenuBarManager)
+    status_bar_manager = Instance(IStatusBarManager)
+    tool_bar_managers = List(IToolBarManager)
```

Comment:
Minor drive-by suggestion
```suggestion
    tool_bar_managers = List(Instance(IToolBarManager))
```